components/inputs/demo/input-select-test.js

```diff
@@ -28,15 +29,17 @@ class TestInputSelect extends RtlMixin(LitElement) {
 	render() {
 		const invalid = this.invalid ? 'true' : 'false';
 		return html`
-			<select
-				aria-label="Choose a dinosaur:"
-				aria-invalid="${invalid}"
-				class="d2l-input-select"
-				?disabled="${this.disabled}">
-				<option>Tyrannosaurus</option>
-				<option>Velociraptor</option>
-				<option>Deinonychus</option>
-			</select>
+			<div class="d2l-skeletize">
```

Comment:
Native `<select>`s don't support `::before` so I needed to do a special workaround that merited a call-out in the docs below.